README.md

```diff
@@ -2,6 +2,8 @@
 
 ## How to Install
 
+## Prerequisite
+- VirtueMart
```

Comment:
Nice of you to update the readme.
Maybe in the prerequisite we can be more descriptive:
- Install [Joomla](https://devilbox.readthedocs.io/en/latest/examples/setup-joomla.html) via Devilbox
- Install [VirtueMart](https://virtuemart.net/downloads) extension or download from Joomla marketplace

Then in line 10, can you update the github release link to: 
https://github.com/xendit/virtuemart-module/releases